cms/tests/test_titlepath.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+from cms.api import create_page, create_title, publish_pages
+from cms.test_utils.testcases import (CMSTestCase)
+
+
+class TitlePathTestCase(CMSTestCase):
+
+    def test0001_translated_subpage_title_path(self):
```

Comment:
Pleae name the tests consistently with the other tests in the project. Maybe: "test_translated_subpage_slug_generation_for_pages_with_different_language_translations"

Reply:
New names are:
"test_translated_subpage_title_path_regeneration"
"test_subpage_title_path_regeneration_after_parent_slug_change"